src/mscorlib/src/System/Diagnostics/Eventing/EventPipe.cs

```diff
@@ -15,12 +15,12 @@ internal struct EventPipeProviderConfiguration
         [MarshalAs(UnmanagedType.LPWStr)]
         private string m_providerName;
         private UInt64 m_keywords;
-        private uint m_loggingLevel;
+        private UInt32 m_loggingLevel;
```

Comment:
Nit: Our convention is to use language keywords instead of BCL types (i.e. int, string, float instead of Int32, String, Single, etc). https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md . So this should have been rather changed other way around.

Reply:
This was changed to fix a marshalling error on x86. Is there perhaps another solution to the marshalling error?

Reply:
I do not see how changing `uint` to `UInt32` can fix marshalling error in this file. They are just different names for the same thing. It is just a matter of style convention which one to use.

Reply:
Same with `long` and `Int64`. These are also different names for the same thing.

Reply:
Here is the commit which fixed the marshalling error. If those are irrelevant, what fixed it?
 Just the changes in the C++ side?
 https://github.com/dotnet/coreclr/pull/14772/commits/97d5c0b1052d761f1dbbe363212cd51cf5f1d958

Reply:
> Just the changes in the C++ side?

I think so.

Reply:
Alright. I'll try reverting the changes to the C# code and test this again